Userland/Libraries/LibUnicode/CharacterTypes.cpp

```diff
@@ -222,6 +222,52 @@ u32 to_unicode_uppercase(u32 code_point)
 #endif
 }
 
+Optional<StringView> code_point_name(u32 code_point)
```

Comment:
Looks like these methods need a [[maybe_unused]] on code_point for the fuzz build. Unless I could tempt you to remove the ENABLE_UNICODE_DATA macro entirely? 👀 https://github.com/SerenityOS/serenity/issues/9466

Reply:
I just also noticed the need for the `[[maybe_unused]]`s ^^

I'm not entirely sure how you would implement this without using ENABLE_UNICODE_DATA?

Reply:
Hmm. I guess you're right. If the Unicode data generator actually ends up linking LibUnicode then you can't get rid of the defines

Reply:
The generators only need definitions from LibUnicode headers so they don't actually link against LibUnicode. The only reason for `ENABLE_UNICODE_DATA` is what you've outlined in #9466 :( (and I guess also for people with bad internet that don't want to download the UCD and CLDR, because they're huge, but idk if anyone actually knows they can disable this).

Reply:
I'm still not entirely sure what this means, but I have a strong feeling that this is out-of-scope for this PR, and I'm not making it any harder, so I'll mark it as resolved :)